fdm-app/app/components/import/changelog1.tsx

```
@@ -0,0 +1,108 @@
+                <div className="mx-auto max-w-3xl">
+                    <h1 className="mb-4 text-3xl font-bold tracking-tight md:text-5xl">
+                        {title}
+                    </h1>
+                    <p className="mb-6 text-base text-muted-foreground md:text-lg">
+                        {description}
+                    </p>
+                </div>
+                <div className="mx-auto mt-16 max-w-3xl space-y-16 md:mt-24 md:space-y-24">
+                    {entries.map((entry, index) => (
+                        <div
+                            key={index}
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)